File: Oppgave.py
#Oppgave 1
#tall1 = float(input("skriv in tall 1: "))
#tall2 = float(input("skriv in tall 2: "))
#operasjon = input("skriv in operasjon (+, -, *, /): ")

#if operasjon == "+":
#    print(tall1 + tall2)
#elif operasjon == "-":
#    print(tall1 - tall2)
#elif operasjon == "*":
#    print(tall1 * tall2)
#elif operasjon == "/":
#    print(tall1 / tall2)
#else:
#    print("Ugyldig operasjon")

#Oppgave 2
#høyde = float(input("skriv in høyde: "))
#bredde = float(input("skriv in bredde: "))

#print("Arealet til rektangelen er", høyde*bredde, "cm")

# Oppgave trey
#navn = input("skriv navn: ")
#alder = int(input("skriv alder (under 100): "))
#
#if alder > 100:
#    print("Ugyldig tall")
#elif alder < 0:
#    print("Ugyldig tall")
#else:
#    print("Hei", navn, ", du fyller 100 om", 100 - alder, "år.")

#Oppgave fiere
#tall = int(input("skriv et tall: "))
#
#if tall > 0:
#    print("Tall", tall, "er et positivt tall")
#elif tall < 0:
#    print("Tall", tall, "er et negativt tall")
#else:
#    print("Tall", tall, "er bare 0 :)")

#Oppgave feim
#tall = int(input("Skriv et tall: "))
#sum = 0
#for i in range(1, tall + 1):
#    sum += i
#
#print(f"Summen av alle tall fra 1 til {tall} er {sum}")

#Oppgave sucks
#navn1 = input("Skriv navn 1: ")
#navn2 = input("Skriv navn 2: ")
#navn3 = input("Skriv navn 3: ")
#navn4 = input("Skriv navn 4: ")
#navn5 = input("Skriv navn 5: ")
#
#navn_liste = [navn1, navn2, navn3, navn4, navn5]
#
#print("Navnene i listen er:", navn_liste)

#Oppgave siuuuuu
navn_liste = []

for i in range(5):
    navn = input(f"Skriv navn {i + 1}: ")
    navn_liste.append(navn)

print("Navnene i listen er:", navn_liste)

File: Oppgave.js
// Oppgave 1
/*
const tall1 = parseFloat(prompt("Skriv inn tall 1: "));
const tall2 = parseFloat(prompt("Skriv inn tall 2: "));
const operasjon = prompt("Skriv inn operasjon (+, -, *, /): ");

if (operasjon === "+") {
    console.log(tall1 + tall2);
} else if (operasjon === "-") {
    console.log(tall1 - tall2);
} else if (operasjon === "*") {
    console.log(tall1 * tall2);
} else if (operasjon === "/") {
    console.log(tall1 / tall2);
} else {
    console.log("Ugyldig operasjon");
}
*/

// Oppgave 2
/*
var høyde = parseInt(prompt("Skriv høyde"));
var bredde = parseInt(prompt("Skriv bredde"));
var areal = (høyde * bredde);

alert(`Arealet til rektangelet er ${areal}`);
*/

// Oppgave trey
/*
const navn = prompt("Skriv navn: ");
const alder = parseInt(prompt("Skriv alder (under 100): "), 10);

if (alder > 100) {
    alert("Ugyldig tall");
} else if (alder < 0) {
    alert("Ugyldig tall");
} else {
    alert(`Hei ${navn}, du fyller 100 om ${100 - alder} år.`);
}
*/

//Oppgave fiere
/*
const tall = parseInt(prompt("Skriv et tall: "), 10);

if (tall > 0) {
    console.log(`Tall ${tall} er et positivt tall`);
} else if (tall < 0) {
    console.log(`Tall ${tall} er et negativt tall`);
} else {
    console.log(`Tall ${tall} er bare 0 :)`);
}
*/

//Oppgave feim
/*
const tall = parseInt(prompt("Skriv et tall: "), 10);

let sum = 0;

for (let i = 1; i <= tall; i++) {
    sum += i;
}

console.log(`Summen av alle tall fra 1 til ${tall} er ${sum}`);
*/

//Oppgave suck
/*
const navn1 = prompt("Skriv navn 1: ");
const navn2 = prompt("Skriv navn 2: ");
const navn3 = prompt("Skriv navn 3: ");
const navn4 = prompt("Skriv navn 4: ");
const navn5 = prompt("Skriv navn 5: ");

const navnListe = [navn1, navn2, navn3, navn4, navn5];

console.log("Navnene i listen er:", navnListe);
*/

//Oppgave siuuuuu
const navnListe = [];

for (let i = 0; i < 5; i++) {
    const navn = prompt(`Skriv navn ${i + 1}: `);
    navnListe.push(navn);
}

console.log("Navnene i listen er:", navnListe);